domains/yrrahbot.acronical.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "AcronicalYT",
+        "email": "acronicalbusiness@gmail.com"
+    },
+    "record": {
+        "CNAME": "9595.site.bot-hosting.net"
```

Comment:
Site seems to be down. Also the preview listed on the PR seems to just show the error instead of the actual website, would like that to get fixed.